### PR TITLE
Updated with off-chain indexing example

### PR DESCRIPTION
**Pull request by jimmychu0807**
Companion PR to: https://github.com/substrate-developer-hub/recipes/pull/435. Merge this only after the recipes PR is merged.

### CODE REVIEW

**Comment by jimmychu0807** [comment]
@sacha-l @NukeManDan This is the companion PR linking to the Recipes off-chain indexing from knowledge base. Should be good to merge.

File: docs/knowledgebase/learn-substrate/off-chain-features.md [hunk 1]
```diff
@@ -104,3 +106,4 @@ refer to the following sections in Substrate Recipes:
 - <span id="example-txs">[Submit signed and unsigned transactions from off-chain workers back on-chain](https://substrate.dev/recipes/off-chain-workers/transactions.html)</span>
 - <span id="example-http">[Fetch external data using HTTP requests and parse JSON responses](https://substrate.dev/recipes/off-chain-workers/http-json.html)</span>
 - <span id="example-off-chain-storage">[Store result in off-chain worker local storage](https://substrate.dev/recipes/off-chain-workers/storage.html)</span>
+- <span id="example-off-chain-indexing">[Example of off-chain indexing](https://substrate.dev/recipes/off-chain-workers/indexing.html)
```
Comment (NukeManDan) [inline docs/knowledgebase/learn-substrate/off-chain-features.md:109]: this is a 404 for me at the moment. Do you mean https://substrate.dev/recipes/off-chain-workers/ ?

**Review by NukeManDan — APPROVED** [review]
once the 404 is fixed, feel free to merge 😁

File: docs/knowledgebase/learn-substrate/off-chain-features.md (hunk 1 above)
Comment (sacha-l) [reply docs/knowledgebase/learn-substrate/off-chain-features.md:109]: Perhaps you meant this? -> https://substrate.dev/docs/en/knowledgebase/learn-substrate/off-chain-features#off-chain-indexing

Comment (jimmychu0807) [reply docs/knowledgebase/learn-substrate/off-chain-features.md:109]: i see. This is due to recipes is not deployed in gh-pages, which is pending on https://github.com/substrate-developer-hub/recipes/issues/438

Comment (jimmychu0807) [reply docs/knowledgebase/learn-substrate/off-chain-features.md:109]: This issue resolved, and Recipes is deployed. Indexing page can be accessed [here](https://substrate.dev/recipes/off-chain-workers/indexing.html).

**Pull request merged**